<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthoriseWindow</class>
 <widget class="QDialog" name="AuthoriseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>87</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttons_layout">
    <item>
     <widget class="QPushButton" name="enter_button">
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close_button">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="name_layout" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="name_box">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="password_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pass_label">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pass_text">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="left_spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="settings_button">
      <property name="text">
       <string>Настройки базы...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pass_text</tabstop>
  <tabstop>enter_button</tabstop>
  <tabstop>close_button</tabstop>
  <tabstop>settings_button</tabstop>
  <tabstop>name_box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
